src/exchangeRateRouter.ts

```diff
@@ -47,6 +48,9 @@ const maybeAddIsoToPair = (pair: string): string =>
 const removeIsoFromPair = (pair: string): string =>
   toIsoPair(subIso, subIso)(fromCode(pair), toCode(pair))
 
+const isIsoPair = (pair: string): boolean =>
+  isIsoCode(fromCode(pair)) || isIsoCode(toCode(pair))
```

Comment:
`iso:ETH_USD` and `ETH_iso:USD` would pass?

Reply:
Yup. It's on the api consumer to use the prefix to identify which is a crypto or fiat code so that it is requested from the appropriate data partner.